src/calculator/votes.ts

```diff
@@ -65,6 +69,14 @@ export async function getVotesWithCoefficients(
     const passportScore = await passportProvider.getScoreByAddress(voter);
     let passportCheckPassed = false;
 
+    if (
+      options.bypassPassportAddresses
+        ?.map((address) => address.toLowerCase())
```

Comment:
Thinking aloud / out of scope for this PR: at some point we might want to use a type to carry information about address validity along with the address itself, since otherwise the only defense against a nasty case bug is our memory.

Reply:
Very relevant thread: https://github.com/ethers-io/ethers.js/discussions/3089

Reply:
I like the second option, the internal value can still technically be messed with but the naming will make it look wrong, it fixes the duck typing issue and there's no Typescript magic to wrap your head around.

Reply:
we could go a step further an name it `__addressDoNotAccessDirectly`, make it look really wrong!

Reply:
moved conversation here 👉 https://github.com/gitcoinco/allo-indexer/issues/324